### PR TITLE
Long press on the map to create an event at that location

### LINKED ISSUE #307: Start Event Creation from long press on the map
Long press on the map should direct the user to the event creation screen with the event location set to the location of the click.

### PR DESCRIPTION
**Pull request by alejandrocalles**
Closes #307.
A long press on the map should direct to the event creation screen.

### CODE REVIEW

**Review by ntissieres — CHANGES_REQUESTED** [review]
The position seems to stick to the "center" of the map which is in Lausanne - EPFL. The coordinates are not good (at least not where the user long click)

File: app/src/main/java/com/github/swent/echo/compose/components/HamburgerMenuDrawerSheet.kt [hunk 1]
```diff
@@ -82,7 +85,10 @@ fun HamburgerMenuDrawerSheet(
             NavigationItem(
                 title = stringResource(id = R.string.hamburger_create_event),
                 selectedIcon = Icons.Filled.AddCircle,
-                navOnClick = { navActions.navigateTo(Routes.CREATE_EVENT) }
+                navOnClick = {
+                    val encodedMapCenter = Json.encodeToString(MAP_CENTER)
```
[the remark this answers is not in the document]
Comment (alejandrocalles) [reply app/src/main/java/com/github/swent/echo/compose/components/HamburgerMenuDrawerSheet.kt:89]: Solved! The logic for displaying the location in `EventLocationEntry` was skewed.

[the remark this answers is not in the document]
Comment (violoncelloCH) [reply app/src/main/java/com/github/swent/echo/compose/components/HamburgerMenuDrawerSheet.kt:89]: great, nice solution!

**Review by violoncelloCH — APPROVED** [review]
great implementation of that quality of life improvement :)
thanks for working on this!
lgtm now

**Pull request merged**